geotools-suite-8.4/pom.xml

```diff
@@ -17,13 +17,13 @@
 
 	<groupId>org.codice.thirdparty</groupId>
 	<artifactId>geotools-suite</artifactId>
-	<version>8.4_3</version>
+	<version>19.1_1</version>
```

Comment:
seems like the directory should be renamed as well

Reply:
Agree. We should remove the version number from the directory